test/transport-test.ts

```diff
@@ -341,10 +341,12 @@ describe('napajs/transport', () => {
 
     let builtinTestGroup = 'Transport built-in objects';
     let nodeVersionMajor = parseInt(process.versions.node.split('.')[0]);
-    if (nodeVersionMajor >= 9) {
+    
+    // TODO #223: Fix transportation support for builtin types, which is broken due to breaking changes of v8 introduced from node v10.0.0. 
+    if (nodeVersionMajor == 9) {
         describe(builtinTestGroup, transportBuiltinObjects);
     } else {
         describe.skip(builtinTestGroup, transportBuiltinObjects);
-        require('npmlog').warn(builtinTestGroup, 'This test group is skipped since it requires node v9.0.0 or above.');
+        require('npmlog').warn(builtinTestGroup, 'This test group is skipped since it requires node v9.0.0 .');
```

Comment:
should be

if (version==10) {
  skip; // because we need to fix 10
} else {
  run_test;
}